src/common/Popover/Popover.js

```diff
@@ -75,17 +86,19 @@ const PopoverController = ({
       const popoverPosition = getPopoverPosition(
         triggerPosition,
         place,
-        offCenter,
         popoverWrapperPosition,
         window.innerWidth
       );
       setStyle(s => ({ ...s, ...popoverPosition }));
     },
-    [setStyle, offCenter, place]
+    [setStyle, place, isOpen]
```

Comment:
Another possibility is to change the PopoverTrigger component, specifically this hook:
    ```
useEffect(() => {
        updatePostion();
    }, [updatePostion, children]);
```
So the position  will be updated after the content is displayed.